### PR TITLE
Update docker-api to 1.28

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by dLobatog** [comment]
I see some files need to be handled by the spec, fixing...

**Comment by domcleal** [comment]
The archive-tar-minitar dependency is now gone, it should be removed from the spec and as a second step, removed from the repository as it's otherwise completely unused:

1. delete the whole directory
2. remove from comps/
3. remove from rel-eng/tito.props
4. don't delete the rel-eng/packages/ file
5. add to tfm/tfm.spec list of obsoletions
6. remove from tfm/scl_prefix_update.conf

**Comment by dLobatog** [comment]
@domcleal Thanks, good catch - I think the latest update contains all these fixes

File: tfm/tfm.spec [hunk 1]
```diff
@@ -65,6 +65,7 @@ Obsoletes: %{scl_prefix}rubygem-sprockets < 2.12.3-3
 Obsoletes: %{scl_prefix}rubygem-less < 2.5.1-5
 Obsoletes: %{scl_prefix}rubygem-less-rails < 2.5.0-4
 Obsoletes: %{scl_prefix}rubygem-uuidtools < 2.1.3-6
+Obsoletes: %{scl_prefix}rubygem-archive-tar-minitar < 0.5.2-11
```
Comment (domcleal) [inline tfm/tfm.spec]: This should be `-12`, as the current release is 11.

Comment (domcleal) [inline tfm/tfm.spec]: Also, please keep the list alphabetical.

**Comment by domcleal** [comment]
Merged and will build shortly, thanks @dLobatog.

**Pull request merged**